docs/links.md

```diff
@@ -31,14 +33,17 @@
 
 ### Synth/Driver
 
-- [OmniMIDI](https://github.com/KeppySoftware/OmniMIDI/) - ⭐ Most optimized for playing Black MIDI. Also works for general players.
+- [OmniMIDI](https://github.com/KeppySoftware/OmniMIDI/) - ⭐ Optimized for playing Black MIDI. Also works for general players.
+- [Xsynth](https://github.com/arduano/xsynth/) - ⭐ Modern MIDI Synth made from the ground up for Black MIDI playback. Supports KDMAPI and built into [Wasabi](https://github.com/arduano/wasabi)
 - [VirtualMIDISynth](https://coolsoft.altervista.org/en/virtualmidisynth) - Was most used before OmniMIDI used, good alternative.
 - [BASSMIDI Player](https://drive.google.com/drive/folders/1yzy9HowL8bao9w--u3VUGIcczF4nrCkI) - Very old driver on which OmniMIDI is based. Very slow also.
 
 ### Tools
 
-- [Keppy's MIDI Converter](https://github.com/KeppySoftware/KMC) - ⭐ Convert MIDI files to MP3, WAV, or OGG.	
+- [Forte](https://github.com/MyBlackMIDIScore/forte) - ⭐ Convert MIDI files to MP3, WAV, or OGG using Xsynth.
+- [Keppy's MIDI Converter](https://github.com/KeppySoftware/KMC) - ⭐ Convert MIDI files to MP3, WAV, or OGG.
```

Comment:
Is the market share of Forte larger than KMC to warrant this?

Reply:
KMC is outdated considered outdated, was replaced by OmniConverter, which isnt even on the list, and even that is considered deprecated. Forte is the most up to date midi to wav converter with all the new features and a lot of people in the BMC server seem to be using it